### PR TITLE
update doc for tikv-ctl

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

File: en/tidb-toolkit.md [hunk 1]
```diff
@@ -92,26 +92,22 @@ pd-ctl -u 127.0.0.1:${local_port} -d config show
         kubectl exec ${pod_name} -n ${namespace} -c tikv -- kill -s TERM 1
         ```
 
-    3. Start the debug container:
+    3. Wait for the TiKV container to restart and enter the container:
```
Comment (ran-huang) [inline en/tidb-toolkit.md:95]:
```suggestion
    3. Wait for the TiKV container to restart, and enter the container:
```
avoid ambiguity.

**Review by ran-huang — APPROVED** [review]

**Pull request merged**